npc/re/merchants/eden_market.txt

```diff
@@ -97,7 +96,21 @@ OnClock0000:
 	npcshopupdate "para_hei10",23016,0,1;
 	npcshopupdate "para_key10",7026,10;
 	npcshopupdate "para_key10",7027,10;
-	npcshopupdate "para_wp00",1181,0,1;
+	npcshopupdate "para_wp00",1181,0,1;	// 2h sword - Tae Goo Lyeon
+	npcshopupdate "para_wp00",1231,0,1;	// MATK dagger - Bazerald 
+	npcshopupdate "para_wp00",13039,0,1;	// ATK dagger - Ivory Knife
+	npcshopupdate "para_wp00",1140,0,1;	// 1h sword - Byeollungum
+	npcshopupdate "para_wp00",1142,0,1;	// 1h spear - Hunting Spear
+	npcshopupdate "para_wp00",1478,0,1;	// 2h spear - Ahlspiess - 2h spear is a tough one, need peer review
+	npcshopupdate "para_wp00",1306,0,1;	// War Axe - 1h axes suck lol
+	npcshopupdate "para_wp00",1137,0,1;	// 2h axe - Hurricane's Fury, other optioon would be Veteran Axe 1384
+	npcshopupdate "para_wp00",16033,0,1;	// Mace - Robot's Mechanical Arm
+	npcshopupdate "para_wp00",1648,0,1;	// 1h staff - Staff of Bordeux - tough choices here, trying to not select a skill/class specific one 
+	npcshopupdate "para_wp00",2004,0,1;	// 2h staff - Kronos
+	npcshopupdate "para_wp00",18117,0,1;	// Royal Bow - this item doesn't drop but it's something different. Other options are Rudra, bow of storms (boring)
+	npcshopupdate "para_wp00",1814,0,1;	// Fist - Berserk. Slotted ver is 1816 which I think should go in instead of non slot (1814). Its not even good
+	npcshopupdate "para_wp00",1584,0,1;	// Book - Cold Magic Book - not many good choices here
+	npcshopupdate "para_wp00",2113,0,1;	// Katar - Bloody Roar
```

Comment:
Currently [Novice Shield](https://www.divine-pride.net/database/item/2113), I believe [Bloody Roar](https://www.divine-pride.net/database/item/1265/bloody-roar) is 1265